### PR TITLE
Chinese version of three docs

### PR DESCRIPTION
**Pull request by rachelzhang0922**
#### Which issue(s) this PR fixes:

Fixes #

#### How is this PR tested:
- [ ] unit test
- [ ] e2e test
- [ ] other (please specify)

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: i18n/zh-Hans/docusaurus-plugin-content-docs/current/guides/stub/http-to-powershell.md [hunk 1]
```diff
@@ -1,10 +1,143 @@
 ---
-title: HTTP to PowerShell Drvier Stub
+title: HTTP 到 PowerShell 的 Stub
```
Comment (tomqin93) [inline i18n/zh-Hans/docusaurus-plugin-content-docs/current/guides/stub/http-to-powershell.md:2]: Stub => 中间件, same for other occurrences

File: i18n/zh-Hans/docusaurus-plugin-content-docs/current/guides/stub/http-to-powershell.md [hunk 2]
```diff
@@ -1,10 +1,143 @@
...
+
+这个HTTP到PowerShell的Stub是这样设计的：
+
+- Stub在主机上暴露了一个HTTP接口
+- 该HTTP接口用于转发来自外部的请求到 `Windows`主机
+- Stub将代理结果和执行状态返回给请求者
+
+### 功能
+
+#### 将HTTP主体代理到PowerShell shell并执行
```
Comment (tomqin93) [inline i18n/zh-Hans/docusaurus-plugin-content-docs/current/guides/stub/http-to-powershell.md:22]: 主体 => 请求体

**Review by tomqin93 — CHANGES_REQUESTED** [review]
@rachelzhang0922 added some comment, please take a look

**Comment by tomqin93** [comment]
@rachelzhang0922 pipeline has failed for this PR, please take a look 
image

File: i18n/zh-Hans/docusaurus-plugin-content-docs/current/guides/stub/http-to-ssh.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
 ---
-title: HTTP to SSH Drvier Stub
+title: HTTP 到 SSH 的 Stub
```
Comment (tomqin93) [inline i18n/zh-Hans/docusaurus-plugin-content-docs/current/guides/stub/http-to-ssh.md:2]: let's keep everything consistent, same for other occurrences 
Stub => 中间件

**Review by tomqin93 — CHANGES_REQUESTED** [review]
@rachelzhang0922 added 1 comment, please take a look

**Comment by Yang-Xijie** [comment]
It will be great to add spaces around English word in Chinese. @rachelzhang0922 

image

I will fix it in this PR soon.

**Comment by Yang-Xijie** [comment]
Please do pay attention to markdown grammar... @rachelzhang0922 Double-check it before you make commits.

image

**Review by tomqin93 — APPROVED** [review]
LGTM

**Pull request merged**